index.js

```diff
@@ -13,7 +13,8 @@ const renderer = remark()
   .use(autolinkHeadings, {behaviour: 'wrap'})
   .use(inlineLinks)
   .use(emoji)
-  .use([hljs, html], {sanitize: false})
+  .use(hljs)
+  .use(html, {sanitize: true})
```

Comment:
Test are failing. Maybe the `[hljs, html]` array needs to be preserved?

Reply:
so you want: `[hljs, html], {sanitize: false}` right?

Reply:
so you want: `[hljs, html], {sanitize: true}` right?